e2e/test/scenarios/admin-2/sso/ldap.cy.spec.js

```diff
@@ -97,16 +97,17 @@ describe(
       cy.findByText("That's not a valid port number").should("exist");
 
       enterLdapPort("21.3");
-      cy.button("Save and enable").click();
-      cy.wait("@updateLdapSettings");
       // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.findByText('For input string: "21.3"').should("exist");
+      cy.findByText("That's not a valid port number").should("exist");
```

Comment:
When we find these disable linter comments, we should see if we can give a more specific selector

Reply:
5d0a519b3d10402f7799f97fba713d9c17da459b